.clang-format

```diff
@@ -0,0 +1,35 @@
+BasedOnStyle: Google
```

Comment:
Remove clang from this PR. You will need to rebase your branch from master as it already has clang format in it.

Reply:
I have a lot to learn.. I just looked up git rebasing

Reply:
This is still in here.